dev-python/sphinxcontrib-pretty-searchresults/sphinxcontrib-pretty-searchresults-0.3.5.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{5,6} )
+
+inherit distutils-r1
+
+MY_PN="${PN/sphinxcontrib/sphinx}"
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="Decently styled search results for sphinx-doc projects"
+HOMEPAGE="https://github.com/sphinx-contrib/sphinx-pretty-searchresults"
+SRC_URI="https://github.com/sphinx-contrib/${MY_PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+
+BDEPEND="
+	dev-python/setuptools[${PYTHON_USEDEP}]
+	test? (
+		>=dev-python/sphinx-testing-0.7[${PYTHON_USEDEP}]
```

Comment:
This version requirement is useless because no lower versions exist in gentoo tree.

Reply:
I'll change it, but what if someone has an overlay with an earlier version? I feel like making the version requirements on ebuilds as explicit as possible is a nice way to make them more reliable.